backend/src/test/java/gov/cdc/usds/simplereport/api/OrganizationFacilityTest.java

```diff
@@ -1,20 +1,20 @@
 package gov.cdc.usds.simplereport.api;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertTrue;
-
-import java.io.IOException;
-import java.util.HashSet;
-import java.util.Set;
+//import static org.junit.jupiter.api.Assertions.assertTrue;
+//
+//import java.io.IOException;
+//import java.util.HashSet;
+//import java.util.Set;
```

Comment:
Seems like we should just remove the unused imports, rather than commenting them out.

If we need to re-add them later, that's pretty trivial.

Reply:
Agreed—commented-out imports are more noisy than helpful.

Reply:
done